modules/dnn/src/darknet/darknet_importer.cpp

```diff
@@ -64,7 +64,7 @@ class DarknetImporter : public Importer
 
 public:
 
-    DarknetImporter() {}
+    DarknetImporter() { net.channels = 0; net.width = 0; net.height = 0; }
```

Comment:
It is better to create initialization ctor for `darknet::NetParameter` (darknet_io.hpp), than initializing its members here.

Reply:
Thank you, I've updated the commit.